aikau/src/test/resources/alfresco/services/ContentServiceTest.js

```diff
@@ -22,9 +22,9 @@
  * @author Dave Draper
  */
 define(["intern!object",
-        "intern/chai!assert",
-        "require",
-        "alfresco/TestCommon"],
+        "intern/chai!assert", 
+        "require", 
+        "alfresco/TestCommon"], 
```

Comment:
I think stripping is fine. My editor seems to have lost its setting for doing so (now restored). Do you need me to push an update with the whitespace gone, or are you happy to merge as is?